docs/products/clickhouse/reference/limitations.rst

```diff
@@ -0,0 +1,92 @@
+Aiven for ClickHouse® limits and limitations
+============================================
+
+This article covers quotas for the Aiven for ClickHouse® service and restrictions on its use. This information helps you make right choices when working with Aiven for ClickHouse, for example, on your infrastructure design, tools, modes of interactions, potential integrations, or handling concurrent workloads.
+
+Limitations
+-----------
+
+From the information about restrictions on using Aiven for ClickHouse, you can easily draw conclusions on how get your service to operate closer to its full potential. Use *Recommended approach* as guidelines on how to work around specific restrictions.
+
+.. list-table::
+   :widths: 25 50 25
+   :header-rows: 1
+
+   * - Name
+     - Description
+     - Recommended approach
+   * - Backups
+     - Aiven for ClickHouse service takes a single snapshot a day. Any data inserted before the next snapshot is be lost when nodes recycle.
```

Comment:
I don't think we should focus too much on what happens when you lose all the nodes of a shards, the point of having multiple copies of the data is to not worry about that.

However there are other consequences of only-daily backups that I think are more relevant to users :

- When powering off the service, all data after the last backup is lost.
- We do not provide point in time recovery, we can only restore the database to the state of one of the daily backups.
- When creating a database fork, you can only create a fork that match the state of one of the backups. 

The first one is the most surprising and the most likely cause sadness if missed. The "normal" meaning of power off does not clearly convey the idea that it can cause data loss. (The console warns about it before validating the power off, but it's worth repeating it in the doc).

In any case, we do not lose data when recycling a node :)

I don't think we talk a lot of "recycling nodes" in the console and docs. We mostly say "apply a maintenance update" or "change the service plan". We go into the details of the nodes only if/when we describe how each operation is implemented.